api/views.py

```diff
@@ -148,10 +148,7 @@ def _formatted_bad_request(format_):
 def enrichment_view(request):
     observable_name = request.query_params.get('query')
     logger.info(f"Enrichment view requested for: {str(observable_name)}")
-    try:
-        required_object = IOC.objects.get(name=observable_name)
-    except IOC.DoesNotExist:
-        return Response({'found':"false"}, status=status.HTTP_200_OK)
+    required_object = EnrichmentSerializer.validate(observable_name)
```

Comment:
this is not the correct way to call the serializer. Did you try running the code ? `validate` is not a class method so this will raise an error.

```python3
serializer = EnchrichmentSerializer(data=request.query_params)
serializer.is_valid(raise_exception=True)
return Response(serializer.data)
```